### PR TITLE
Table for alg param - QGIS Cartography

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 1]
```diff
@@ -20,23 +20,58 @@ databases these will be renamed to have unique names in the output combined data
...
+   :stub-columns: 0
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Input databases**
+     - ``INPUT``
+     - [file] [list]
+     - Input line or polygon vector layer
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/cartography.rst]:
```suggestion
     - Files containing QGIS style items
```

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 2]
```diff
@@ -20,23 +20,58 @@ databases these will be renamed to have unique names in the output combined data
...
+   * - **Objects to combine**
+     - ``OBJECTS``
+     - [enumeration] [list]
+     - Files containing QGIS style items. One or more of:
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/cartography.rst]:
```suggestion
     - Types of style items in the input databases you would like to
       put in the new database.  These can be:
```

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 3]
```diff
@@ -20,23 +20,58 @@ databases these will be renamed to have unique names in the output combined data
...
+       * 1 --- :ref:`Color ramps <color-ramp>`
+       * 2 --- :ref:`Text formats <text_format>`
+       * 3 --- :ref:`Label settings <showlabels>`
+
+   * - **Output style database**
+     - ``OBJECTS``
+     - [file]
+       
+       Default: ``[Save to temporary file]``
+     - Specify the output (bounding box) layer.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/cartography.rst]:
```suggestion
     - Output :file:`.XML` file combining the selected style items
```

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 4]
```diff
@@ -114,23 +214,76 @@ dialog.
...
+   * - **Input project (leave blank to use current)**
+
+       Optional
+     - ``INPUT``
+     - [file]
+     - A QGIS project file to extract the style items from
+   * - **Objects to extract**
+     - ``OBJECTS``
+     - [enumeration] [list]
+     - Files containing QGIS style items. One or more of:
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]:
```suggestion
     - Types of style items in the input project you would like to
       put in the new database. These can be:
```

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 5]
```diff
@@ -114,23 +214,76 @@ dialog.
...
+       * 1 --- :ref:`Color ramps <color-ramp>`
+       * 2 --- :ref:`Text formats <text_format>`
+       * 3 --- :ref:`Label settings <showlabels>`
+
+   * - **Output style database**
+     - ``OUTPUT``
+     - [file]
+
+       Default: ``[Save to temporary file]``
+     - Specify the output (bounding box) layer.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/cartography.rst]:
```suggestion
     - Specify the output :file:`.XML` file for the selected style items
```

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 6]
```diff
@@ -169,11 +314,92 @@ Parameters
...
+       Default: *All the map items*
+     - The map item(s) whose information you want to
+       extract.
+       If none is provided then all the map items
+       are processed.
+   * - **Overrride CRS**
+
+       Optional
+     - ``CRS``
+     - [enumeration] [list]
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Will fix.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 7]
```diff
@@ -169,11 +314,92 @@ Parameters
...
+     - Type
+     - Description
+   * - **Map heigh**
+     - ``HEIGHT``
+     - [number]
+     - 
+   * - **Extent**
+     - ``OUTPUT``
+     - [vector: polygon]
+     - Output polygon vector layer containing the input layout extent
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Will try to look into this.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 8]
```diff
@@ -151,16 +304,8 @@ the original map item CRS will be used.
...
-``Map item`` [enumeration]
-  Optional
-
-  The map item(s) whose information you want to extract.
-  If none is provided then all the map items are processed.
-
-  Default: *All the map items*
+  
 
 ``Override CRS`` [crs]
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Will fix.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Right! Fixed.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Fixed (copy paste...)

File: source/docs/user_manual/processing_algs/qgis/cartography.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Thanks. Fixed.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 9]
```diff
@@ -58,44 +93,109 @@ could not be matched to symbols, and symbols which were not matched to categorie
...
+       The file encoding can also be changed here.
+   * - **Non-matching symbol names**
+
+       Optional
+     - ``NON_MATCHING_SYMBOLS``
+     - [table]
+
+       Default: ``[Skip output]``
+     - Output table for symbols from the provided style database which
+       does not match any category. One of:
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Thanks.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 10]
```diff
@@ -58,44 +93,109 @@ could not be matched to symbols, and symbols which were not matched to categorie
...
+       categories and symbols names will be ignored, allowing
+       greater tolerance during the match.
+   * - **Non-matching categories**
+
+       Optional
+     - ``NON_MATCHING_CATEGORIES``
+     - [table]
+
+       Default: ``[Skip output]``
+     - Output table for categories which does not match any symbol in
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Thanks.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 11]
```diff
@@ -169,11 +314,92 @@ Parameters
...
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Print layout**
+     - ``LAYOUT``
+     - [enumeration]
+     - A print layout in the current project
+   * - **`Map item**
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Thanks.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 12]
```diff
@@ -169,11 +314,92 @@ Parameters
...
+.. list-table::
+   :header-rows: 1
+   :widths: 20 20 20 40
+   :stub-columns: 0
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Map heigh**
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Thanks. Fixed.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 13]
```diff
@@ -199,47 +425,96 @@ can see each color class has the same amount of features.
...
-    balanced. This mode can be useful to help avoid large features resulting in
-    one of the colors appearing more dominant on a colored map.
-
-
-  * 2 --- By distance between colors
-
-    Assigns colors in order to maximize the distance between features of the same
-    color. This mode helps to create a more uniform distribution of colors across
-    a map.
+Parameters
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Thanks.

**Comment by havatv** [comment]
Thanks for reviewing, @DelazJ.  Far too many cut and paste errors here (and too much uncritical copying of the existing documentation). I hope this is in better shape now.

File: source/docs/user_manual/processing_algs/qgis/cartography.rst [hunk 14]
```diff
@@ -148,32 +319,88 @@ be exported. If it is not specified, all map extents from the layout will be exp
...
+     - Type
+     - Description
+   * - **Map height**
+     - ``HEIGHT``
+     - [number]
+     - 
+   * - **Extent**
+     - ``OUTPUT``
+     - [vector: polygon]
+     - Output polygon vector layer containing an extent that covers all
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/cartography.rst]: It's not an extent feature covering all the map items; it's an extent feature for each map item

Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Better now?
`Output polygon vector layer containing an extents of all the input layout map item(s)`

Comment (DelazJ) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: without the "**an**"
Thanks

Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/cartography.rst]: Thanks. Removed.

**Pull request merged**